<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSearchBase</class>
 <widget class="QWidget" name="SSearchBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSearchBase</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="clearqPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Erase search entry</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Start query</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="searchTypCMB">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Choose search type.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="queryText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>8</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>155</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Enter search terms here. Type ESC SPC for completions of current term.</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>200</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <resources/>
 <connections/>
</ui>
